.gitignore

```diff
@@ -28,6 +28,7 @@ modules/
 spec/fixtures/
 .vagrant
 coverage/
+.rake_tasks~
```

Comment:
Should this have a ~?

Reply:
this is fine. Just making sure. Some editors like vi use ~ for line endings and it gets pasted by accident.

Reply:
So, no changes needed? Ready for merging?  :smiley: :crossed_fingers: